aws/resource_aws_ec2_client_vpn_network_association.go

```diff
@@ -29,14 +30,19 @@ func resourceAwsEc2ClientVpnNetworkAssociation() *schema.Resource {
 			},
 			"security_groups": {
```

Comment:
This attribute should be updated to include the following:

* `MinItems: 1` - it is not valid to configure the target network without at least one security group. It defaults to the VPC default security group (which cannot be deleted).
* `MaxItems: 5` - Max according to the [EC2 API Reference](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ApplySecurityGroupsToClientVpnTargetNetwork.html)